bundles/org.openhab.binding.energidataservice/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="energidataservice"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="spot-price">
+		<item-type>Number</item-type>
+		<label>Spot Price</label>
+		<description>Spot price.</description>
+		<category>Price</category>
+		<state readOnly="true" pattern="%.9f"></state>
```

Comment:
Is 9 positions after the decimal point correct? Same for below (6).

Reply:
Yes, it's correct.